docs/polaris/containers/containers.md

```diff
@@ -114,18 +114,21 @@ The latest containers are updated periodically. If you have trouble using contai
  
 
 ## Troubleshooting Common Issues
-1. **Permission Denied Error**: If you encounter permission errors during the build
+
+- **Permission Denied Error**: If you encounter permission errors during the build
 	- Check your quota and delete any unnecessary files. 
-	- Clean-up apptainer cache, `~/.apptainer/cache`, and set the apptainer tmp and cache directories as below:
+	- Clean-up apptainer cache, `~/.apptainer/cache`, and set the apptainer tmp and cache directories as below. If your home directory is full and if you are building your container on a compute node, then set the tmpdir and cachedir to local scratch 
```

Comment:
```suggestion
	- Clean-up Apptainer cache, `~/.apptainer/cache`, and set the Apptainer temporary and cache directories as below. If your home directory is full and if you are building your container on a compute node, then set the temporary and cache directories to local scratch 
```

I would recommend using "Apptainer", capitalized whenever referring to the project/concept outside of a code block, as the project does in their own literature: https://github.com/apptainer/apptainer
I think it makes our docs look more polished

Reply:
also earlier in this file: 
> This is important for system (Polaris - Cray) mpich to bind to containers mpich. Set the following environment variables

to 

> This is important for the system's Cray MPICH distribution to bind to the MPICH installation inside a container. Set the following environment variables